src/object.d

```diff
@@ -1689,7 +1689,6 @@ class Throwable : Object
         sink("@"); sink(file);
         sink("("); sink(sizeToTempString(line, tmpBuff, 10)); sink(")");
 
-        auto msg = message();
```

Comment:
BTW, this wouldn't work if `message()` would allocate. We use the sink based toString to print exception messages when the GC cannot be used.